src/components/forms/VsiForm.js

```diff
@@ -57,6 +59,16 @@ class VsiForm extends Component {
     let classNameModalCheck = this.props.isModal
       ? "fieldWidthSmaller"
       : "fieldWidth";
+    let vsiVolumeInnerFormProps = {
+      invalidCallback: this.props.invalidVsiVolumeCallback,
+      invalidTextCallback: this.props.invalidVsiVolumeTextCallback,
+      composedNameCallback: this.props.composedNameCallback,
```

Comment:
remove composed name callback, not needed. do not use helper text for name of block storage volume